src/components/Scoreboard.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { formatDuration } from '../utils';
+import { Navigation } from './Navigation';
+
+export const Scoreboard = () => {
+  const [pastScores, setPastScores] = useState([])
+  useEffect(() => {
```

Comment:
Nope! :D Komponent nie powinien zawierać logiki a tym bardziej bić do API to za dużo, tu może być cieżko nie zawierać jakieś logiki ale chociaż wydziel tego call do API. Tutaj powinna być dispachowana akcja, która dopiero będzie pobierała dane i zapisywała je do redux, tutaj całkowicie ominełaś ten etap